### ISSUE TITLE
[Feature request]

### ISSUE THREAD

**Issue by lazerleif** [OmGodse/Notally#20]
Hey, first of all love the app! There are really only two things i miss. Posted another issue aswell

If i copy a list for example from a SMS and paste it in a list there will only be one line with one checkbox but several lines in that single post.
Would be great if you could fix implement this 😄

**Comment by OmGodse** [comment]
Could you please attach an image of the type of list you want to copy from an SMS (for some clarity) ?

**Comment by lazerleif** [comment]
Of course, its just linebreaks

![Screenshot_20200706-170219_Textra](https://user-images.githubusercontent.com/37516628/86608327-e4568b80-bf99-11ea-828f-39b1235b5b51.png)

**Comment by moriel5** [comment]
May I please add to this request?

I agree that it would be preferable to add separate lines as separate tasks, however there situations where I need multiple lines for a single task.

For such situations, could the enter key output an additional line, with a button within Notally to add another task to the list?

**Comment by Erz3** [comment]
> May I please add to this request?
> 
> I agree that it would be preferable to add separate lines as separate tasks, however there situations where I need multiple lines for a single task.
> 
> For such situations, could the enter key output an additional line, with a button within Notally to add another task to the list?

Yes, would be more intuitive if you could add a "+" icond to get another check-box.
